### ISSUE TITLE
Each A tag must contain text or an IMG with an ALT attribute

### ISSUE THREAD

**Issue by hortongn** [uclibs/scholar_uc#515]
Add text to the link, or ALT text if the link contains an image. If there is no link text or the ALT tag is blank, screen readers have nothing to read, so read out the URL instead.		

Problem pages:
* https://scholar-qa.uc.edu/catalog
* https://scholar-qa.uc.edu/catalog?f%5Bgeneric_type_sim%5D%5B%5D=Collection
* https://scholar-qa.uc.edu/catalog?f%5Bgeneric_type_sim%5D%5B%5D=Work
* https://scholar-qa.uc.edu/catalog?f%5Bhuman_readable_type_sim%5D%5B%5D=Article
* https://scholar-qa.uc.edu/catalog?f%5Bhuman_readable_type_sim%5D%5B%5D=Collection	
* https://scholar-qa.uc.edu/catalog?f%5Bhuman_readable_type_sim%5D%5B%5D=Document
* https://scholar-qa.uc.edu/catalog?f%5Bhuman_readable_type_sim%5D%5B%5D=Image	
* https://scholar-qa.uc.edu/catalog?page=2	
* https://scholar-qa.uc.edu/catalog?page=3
* https://scholar-qa.uc.edu/catalog?page=4

**Comment by lisa3711** [comment]
Looked this over- cant find any images without alt text. On the navbar tabs - the current tab does not have a title, looking into using sr- or title attributes.

**Comment by lisa3711** [comment]
I cannot find any images on Scholar 2.X without alt-text. Closing
